Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -86,7 +90,13 @@ def bump_formula_pr
 
     old_formula_version = formula_version(formula, requested_spec)
 
-    replacement_pairs = if new_url_hash
+    replacement_pairs = if requested_spec == :stable && formula.revision != 0
+      [[/^  revision \d+\n/m, ""]]
+    else
+      []
+    end
```

Comment:
Will probably be somewhat clearer (and shorter) like this:

```ruby
replacement_pairs = []
if requested_spec == :stable && formula.revision != 0
  replacement_pairs << [/^  revision \d+\n/m, ""]
end
```

(Or move that below the mandatory replacements, so you won't even have to initialize the variable.)